### DISCUSSION TITLE
Odin resetting Valheim Plus

### DISCUSSION THREAD

**Discussion by XBott94** [mbround18/valheim-docker#313] [Help!] [answered]
On container restart Odin reinstalls Valheim Plus and so resets the config file.
Can I somehow configure the container to not reinstall Valheim Plus everytime, or can I somehow reload the modified config while the container is running?

**Comment by CosmicHorrorDev** [comment, marked as answer]
There was a fix pushed through a coue days ago that should keep old config files now
